.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,32 @@
+### Changes
```

Comment:
I don't know why we call this section `changes` it should be `description` or something like that. Changes implies what changed, which can be gathered from the diff. There should be a broad description of why this pull request is even needed (what problem are we solving) and the approach to that problem

Reply:
Probably what would be useful here instead of just turning a diff into a description would be an information if we are introducing any *breaking changes*. That would further engage a new version release, documentation changes etc., that could be useful.

Reply:
Yes, agreed!